structure/0006-governance.md

```diff
@@ -2,53 +2,152 @@
+
+After the voting period concludes, the EOB may take up to one week to count the votes. In the event of a tie, votes by existing admin board members will count as three votes. The EOB may privately inform the winning candidate, before publishing the results to the organization members and subsequently the community at large.
+
+## Processes
+
+To streamline working with teams, some standard processes need to be defined.
+The following processes apply to all teams unless that team's RFC specifies a different process.
+
+### Voting
+
+Unless specified otherwise, the following process is used for voting:
+
+A vote in a team may be called at any time by a team lead for that team or for one of its parent teams.
+A vote can also be initiated as required by another process, such as adding or removing a team member. 
+
+When a vote is called within a team, all members of that team and its parent teams are eligible to vote.
+All polls should include a positive and negative option, as well as the option to abstain.
+Voting members that abstain are excluded from percentages and tallies.
+Voting members not casting a vote within the allotted time frame are counted as abstained.
+
+After one week, the plurality of votes wins. The vote may also be called already if all eligible members 
+casted their vote.
+
+### Introduction of Staff
+
+Any community member is eligible to be added to a team by a majority vote of all eligible organization members.
+The vote is started on request from an eligible organization member. A community member wishing to join a team may
+kindly ask an eligible organization member to request a vote. A positive response from the organization member isn't
+mandatory.
+The following members are eligible:
+- All members of the relevant team
+- All members of all the sub teams of the relevant team
+- All members of all the parent teams of the relevant team
+
+During the vote, especially if the community member isn't already part of any team, the rest of the organization is
+consulted for opinions.
+
+### Removal of Staff
+
+Any team member or administrative board member may be removed by a majority vote of all eligible organization members.
+The following members are eligible:
+- All members of the relevant team
+- All members of all the sub teams of the relevant team
+- All members of all the parent teams of the relevant team
```

Comment:
It'd probably be good to define this list as "all teams" where the admin board is concerned.